core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -60,6 +60,16 @@
 import nl.nn.adapterframework.util.RunStateManager;
 
 /**
+ * An Adapter receives a specific type of messages and processes them. It has {@link Receiver}-s
+ * that determine how messages are received and a {@link PipeLine} that determines how
+ * the messages are processed. If an adapter can receive its messages from multiple sources,
```

Comment:
Het is niet de receiver/de pipeline die het 'bepaalt', maar de configuratie.

Reply:
Verwerkt. Ik heb het over wat elementen doen, niet wat ze bepalen.